clients/cli/src/main/java/org/apache/gravitino/cli/CommandLineOutputs.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.cli;
+
+import org.apache.commons.cli.Option;
+import org.apache.gravitino.cli.outputs.StringFormat;
+import org.apache.gravitino.cli.outputs.TableFormat;
+
+/** Outputs different format for the CLI results. */
+public class CommandLineOutputs {
+  public static <T> void output(T entity) {
+    if (GravitinoCommandLine.getOptions().hasOption("output")) {
```

Comment:
A test like this is unnecessary.
The `getOption()` method returns null if there is no such an option.